source/PlayerInfo.cpp

```diff
@@ -2820,6 +2821,11 @@ void PlayerInfo::Save(const string &path) const
 	DataWriter out(path);
 
 
+	// Current game version:
+	string version = Format::Split(Format::Split(Files::Read(Files::Resources() + "credits.txt"), "\n")[1], " ")[1];
+	version = version.substr(0, version.length() - 1);
+	out.Write("version", version);
```

Comment:
I would like the version, and the Git-commit-ID to be written out as 2 separate items.

Reply:
This comment still seems open; I would like to see the version-string and the commit-ID to be written under 2 separate keywords in the savegame.
Having 2 separate keywords makes it easier to perform automated actions on this information, if we choose to add such automated actions in the future.

Reply:
Do we even need the commit hash? The version number ought to be enough no? Although I can't think of any automated actions we could do with this, except for showing a warning if you open a save game from a later version of the game.

Reply:
@quyykk I imagine quite some people compile from source, so for those you need commit hashes to figure out what they're running exactly

Reply:
Should be done now. Note that, for people building from source themselves, the commit hash won't actually be present, but the continuous release should have it.